app/views/clients/createclient.html

```diff
@@ -265,7 +265,7 @@
                      <div ng-repeat="addr in addressArray">
                         <div class="form-group" >
                            <div class="control-label col-sm-2">
-                              <label>{{ 'label.input.addressType' | translate }}</label>
+                              <label>{{ 'label.input.addressType' | translate }}<span class="required">*</span></label>
                            </div>
                            <div class="col-sm-3"  >
                               <select chosen="addressTypes"  ng-model="addressArray[$index].addressTypeId"  class="form-control"
```

Comment:
Need to add required and late-validate attributes, just adding * won't fix this.